Maestro/subscriptions.json

```diff
@@ -416,6 +416,35 @@
         }
       }
     },
+    // Update dependencies in CoreCLR release/3.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/optimization/release/3.0/Latest.txt"
+      ],
+      "action": "coreclr-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoSourceBranch": "release/3.0",
+        "vsoBuildParameters": {
+          "ScriptFileName": "run.cmd",
+          "Arguments": [
+            "build",
+            "-Project='tests\\build.proj'",
+            "--",
+            "/t:UpdateDependenciesAndSubmitPullRequest",
+            "/p:GitHubUser=dotnet-maestro-bot",
+            "/p:GitHubEmail=dotnet-maestro-bot@microsoft.com",
+            "/p:GitHubAuthToken=`$(`$Secrets['DotNetMaestroBotGitHubToken'])",
+            "/p:ProjectRepoOwner=dotnet",
+            "/p:ProjectRepoName=coreclr",
+            "/p:ProjectRepoBranch=release/3.0",
+            "/p:NotifyGitHubUsers=dotnet/maestro-reviewers-core",
+            "/verbosity:Normal"
```

Comment:
CoreCLR recently had to change this because they removed run.cmd:

https://github.com/dotnet/versions/blob/57bb1de0d2f0a518c60a228fca6911938f5ee21d/Maestro/subscriptions.json#L172-L185

Seeing this, I think the 3.0 CoreFX sub needs to be changed too...

(Core-Setup hasn't changed, though. Hasn't switched to Arcade SDK yet.)

Reply:
@dagood fixed, does it look good now?

Reply:
CoreCLR looks ok now, here's what CoreFX looks like in master though:

https://github.com/dotnet/versions/blob/52912e7390f2783de5f60810715274c20c5307df/Maestro/subscriptions.json#L463-L473

Reply:
Fixed